docs/developer/permissions.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: Permissions
+sidebar_label: Permissions
+---
+
+## User permissions
+
+Its group defines user permissions. Organizing access rights into [Groups](developer/api-reference/objects/group.mdx) helps in determining staff users' roles in the organization.
```

Comment:
"Its group defines user permissions." - What do you mean exactly? I'm not sure I understand this sentence.

Reply:
You cannot assign permissions to the user directly, only by assigning user to permission group

Reply:
Ah so this makes it more clear, maybe the explanation you gave should be included in the docs instead?

Reply:
```suggestion
Instead of assigning permissions directly to the user, we define them on a by-group basis. Organizing access rights into [Groups](developer/api-reference/objects/group.mdx) helps in determining staff users' roles in the organization.
```